addons/hr_contract/views/hr_contract_views.xml

```diff
@@ -143,7 +143,7 @@
         <record id="hr_contract_view_form" model="ir.ui.view">
             <field name="name">hr.contract.form</field>
             <field name="model">hr.contract</field>
-            <field name="arch" type="xml">
+           <field name="arch" type="xml">
```

Comment:
I guess this is a oupsi?

Reply:
this needs fixing and also commit header :innocent: 

It should be just `[IMP] hr_contract:`

Reply:
applied